pageserver/src/walreceiver/walreceiver_connection.rs

```diff
@@ -116,30 +129,11 @@ pub async fn handle_walreceiver_connection(
 
     let end_of_wal = Lsn::from(u64::from(identify.xlogpos));
     let mut caught_up = false;
-    let ZTenantTimelineId {
-        tenant_id,
-        timeline_id,
-    } = id;
-
-    connection_status.latest_connection_update = Utc::now().naive_utc();
-    connection_status.latest_wal_update = Utc::now().naive_utc();
-    connection_status.commit_lsn = Some(end_of_wal);
-    if let Err(e) = events_sender.send(TaskEvent::NewEvent(connection_status.clone())) {
```

Comment:
Should we retain the pinging after IDENTIFY_SYSTEM?

Reply:
restored

Reply:
I'm not sure if it's needed, but if we decide to remove it, that should be a separate PR. It wasn't intentional here.